CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Bug fixes
 - Fix errata in the voc document. Color values in the labelmap.txt should be separated by commas, not colons.
   (<https://github.com/openvinotoolkit/datumaro/pull/1162>)
+- Fix hyperlink errors in the document
```

Comment:
Is this right place to log changes for 1.5.1?

Reply:
I think so.